runtime/imported_values_memory_metering_test.go

```diff
@@ -396,7 +396,7 @@ func TestImportedValueMemoryMetering(t *testing.T) {
 
 		executeScript(t, script, meter, structValue)
 		assert.Equal(t, uint64(1), meter[common.MemoryKindCompositeValueBase])
-		assert.Equal(t, uint64(71), meter[common.MemoryKindRawString])
+		assert.Equal(t, uint64(284), meter[common.MemoryKindRawString])
```

Comment:
Hmm, this is a result of removing the type ID caching. I might have to revert the changes and fix it on master instead

Reply:
In #2751 I improved composite and interface static types on master in a different way, by adding type ID caching to interface static types, instead of removing it from composite static types.

I'll revert the changes here once that PR is merged.